server/src/main/java/org/apache/iotdb/db/mpp/common/header/ColumnHeaderConstant.java

```diff
@@ -68,6 +68,8 @@ public class ColumnHeaderConstant {
   public static final String COLUMN_STATUS = "Status";
   public static final String COLUMN_HOST = "Host";
   public static final String COLUMN_PORT = "Port";
+  public static final String COLUMN_INTERNAL_PORT = "InternalPort";
+  public static final String COLUMN_RPC_PORT = "RpcPort";
```

Comment:
Is the static field COLUMN_PORT can be deleted now?

Reply:
delete